### DISCUSSION TITLE
Scroll within content

### DISCUSSION THREAD

**Discussion by antistic** [plateaukao/einkbro#187] [Ideas]
The scroll buttons only work to scroll the main body, not any elements inside the body. This is inconvenient on sites where the main content isn't in the main element, such as [feedbin](https://feedbin.com/), [instapaper](https://www.instapaper.com/), [gmail](mail.google.com), [inoreader](https://www.inoreader.com/).

**Comment by plateaukao** [comment]
For these sites, I don't know what I can do to make it work.
